### PR TITLE
Accompanying change for s3mpi change to support safe = TRUE/FALSE

### PR DESCRIPTION
**Pull request by kirillseva**
Bumped the package version so that we can later enforce it using the lockfile

### CODE REVIEW

File: R/adapter.r [hunk 1]
```diff
@@ -245,9 +245,15 @@ construct_s3_adapter <- function() {
...
+    # If the user provided an s3 path, like "s3://somebucket/some/path/",
+    # or wants to overwrite an already existing file by providing an option,
     # pass it along to the s3read function.
-    args <- list(obj = object, name = opts$resource)
+    args <- list(obj = object, name = opts$resource, safe = FALSE)
+    # Set `safe = FALSE` by default for backwards compatibility.
+    # Since running syberia files is a reproducible process there
+    # shouldn't be a lot of damage done
+    # There is an option to overwrite this behavior
+    if (is.element('safe', names(opts))) args$safe <- opts$safe
```
Comment (kirillseva) [inline R/adapter.r]: this is the non-trailing-whitespace change

File: DESCRIPTION [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Package: syberiaStages
 Type: Package
 Title: Syberia Stages
-Version: 0.2.1
+Version: 0.3.0
```
Comment (robertzk) [inline DESCRIPTION]: 0.2.2 please

File: R/adapter.r [hunk 3]
```diff
@@ -286,12 +292,12 @@ construct_s3data_adapter <- function() {
   write_function <- function(object, opts) {
     common_s3mpi_package_loader()
 
-    obj <- list(data = switch(1 + is.element("data", names(object$output$options)), 
-                              NULL, object$output$options$data), 
-                label = switch(1 + is.element("label", names(object$output$options)), 
+    obj <- list(data = switch(1 + is.element("data", names(object$output$options)),
```
Comment (robertzk) [inline R/adapter.r]: I would avoid the `switch` statement in R in almost all situations. It's slower and less readable. Your code is equivalent to:

```r
object <- list(data = if (is.element("data", names(try(silent = TRUE, object$output$options))) object$output$options$data)
```

Make sure you don't reference `object$output$options` without a `try` or you will break this adapter if people use it for anything except a tundraContainer.

File: R/adapter.r [hunk 4]
```diff
@@ -286,12 +292,12 @@ construct_s3data_adapter <- function() {
...
+                              NULL, object$output$options$data),
+                label = switch(1 + is.element("label", names(object$output$options)),
```
Comment (robertzk) [inline R/adapter.r]: Same here.

File: R/adapter.r (hunk 3 above)
Comment (kirillseva) [inline R/adapter.r]: git blame
Git blame :haircut:

File: R/adapter.r (hunk 1 above)
Comment (robertzk) [inline R/adapter.r]: Why not 

```r
args$safe <- opts$safe %||% args$safe
```

**Comment by robertzk** [comment]
Looks good!

File: R/adapter.r (hunk 1 above)
Comment (kirillseva) [inline R/adapter.r]: Just to have the same style in two adjacent lines
```
if (is.element('safe', names(opts))) args$safe <- opts$safe
if (is.element('s3path', names(opts))) args$.path <- opts$s3path
```
Should I refactor both before merge?

Comment (robertzk) [inline R/adapter.r]: Yes please, I don't know what I was thinking.

**Pull request merged**